packages/fluentui/react-northstar/src/themes/teams/components/Tree/treeTitleVariables.ts

```diff
@@ -1,13 +1,52 @@
 import { pxToRem } from '../../../../utils';
+import * as _ from 'lodash';
 
 export interface TreeTitleVariables {
   color: string;
+  indicatorColor: string;
+
   padding: string;
+  gap: string;
+  selectionIndicatorMargin: string;
+  borderColor: string;
+  borderStyle: string;
+  borderRadius: string;
+  borderWidth: string;
+  background: string;
+
+  checkedBackground: string;
+  checkedBorderColor: string;
+  checkedIndicatorColor: string;
+
+  disabledBackground: string;
+  disabledBackgroundChecked: string;
+  disabledBorderColor: string;
+  disabledCheckedIndicatorColor: string;
 }
 
+const defaultValue = 'red';
+
 export default (siteVars: any): TreeTitleVariables => {
   return {
     color: siteVars.colorScheme.default.foreground,
     padding: `${pxToRem(1)} 0`,
+
+    borderColor: _.get(siteVars, 'colorScheme.default.foreground1', defaultValue),
```

Comment:
```suggestion
    borderColor: siteVars.colorScheme.default.foreground1,
```

Let's omit Lodash here. I don't obvious reasons to use it here

Reply:
fixed, removed :) it was caused by copy from checkbox